components/decentralized_dns/decentralized_dns_opt_in_page.h

```diff
@@ -9,6 +9,7 @@
 #include <memory>
 #include <string>
 
+#include "base/values.h"
```

Comment:
could be a forward decl?

Reply:
Fixed in https://github.com/brave/brave-core/commit/c4fc3b521dd321373d7e46222752410bb9357e0f